src/pluginutils.c

```diff
@@ -554,6 +554,24 @@ void plugin_builder_connect_signals(GeanyPlugin *plugin,
 }
 
 
+/** Get the additional data that corresponds to the plugin.
+ *
+ * @param plugin The plugin provided by Geany
+ * @return The data corresponding to the plugin or @c NULL if none set.
+ *
+ * @since 1.29 (API 230)
+ *
+ * @see geany_plugin_set_data()
+ */
+gpointer geany_plugin_get_data(const GeanyPlugin *plugin)
+{
+	g_return_val_if_fail (plugin != NULL, NULL);
+	g_return_val_if_fail (PLUGIN_LOADED_OK (plugin->priv), NULL);
```

Comment:
These should be normal `if(..)return val` because they should not be removable by compile option as the `g_...` checks are.